### PR TITLE
tools: Add option to write normals to mesh_sampling

### PR DESCRIPTION
**Pull request by MarcoFalke**
This should be relatively easy to review, as it should work with any obj/ply file.

Example before:
screenshot from 2017-01-19 02-40-40

Example after:
screenshot from 2017-01-19 02-40-37

### CODE REVIEW

**Comment by MarcoFalke** [comment]
(Added screenshots for reference)

This should be easy to test if a newcomer wants to give it a try...

File: tools/mesh_sampling.cpp [hunk 1]
```diff
@@ -230,30 +247,43 @@ main (int argc, char **argv)
...
-  savePCDFileASCII (argv[pcd_file_indices[0]], *res);
+  if(!write_normals)
+  {
+    pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_xyz (new pcl::PointCloud<pcl::PointXYZ>);
+    pcl::copyPointCloud(*cloud_1, *cloud_xyz);
+    savePCDFileASCII(argv[pcd_file_indices[0]], *cloud_xyz);
+  }
+  else
+  {
+    savePCDFileASCII(argv[pcd_file_indices[0]], *res);
```
Comment (taketwo) [inline tools/mesh_sampling.cpp]: What is the difference between `res` and `cloud_1`?

Comment (MarcoFalke) [inline tools/mesh_sampling.cpp]: Good catch!

`res` is the voxelized cloud. I should fix my typo and maybe rename `res` to something more meaningful.

Comment (taketwo) [inline tools/mesh_sampling.cpp]: Since you'll be updating your commit anyway, please insert spaces between function names and braces to match PCL style.

**Comment by taketwo** [comment]
Looks good to me, thanks!

**Pull request merged**